Jenkinsfile

```diff
@@ -71,7 +71,7 @@ pipeline {
         }
     }
     parameters {
-        string(defaultValue: "5.0.5",
+        string(defaultValue: "5.2.0",
```

Comment:
This change could be done in a separate PR and be merged much faster.

Reply:
moved to https://github.com/scylladb/scylla-cluster-tests/pull/6179